### ISSUE TITLE
Some parser false positives

### ISSUE THREAD

**Issue by a-x-** [csstree/csstree#17]
As I know followed case are false positive

```
Mismatch
  syntax: [ <bg-layer> , ]* <final-bg-layer>
  background: linear-gradient(left,rgba(0,0,0,0) 0,rgba(0,0,0,.2) 50%,rgba(0,0,0,0) 100%)
  ------------^
Mismatch
  syntax: none | <transform-list>
  transform: rotate(0)
  -----------^
Unknown property
  tap-highlight-color: transparent
  ^
Unknown property
  -webkit-print-color-adjust: exact
  ^
Uncomplete match
  syntax: <single-transition>#
  transition: visibility 0 .3s, opacity .3s ease-out
  -----------------------^
Mismatch
  syntax: [ <bg-layer> , ]* <final-bg-layer>
  background: linear-gradient(top,#fff,#f2f2f2)
  ------------^
```

**Comment by lahmatiy** [comment]
@a-x- Thank you for your feedback!

Your gradient values are wrong since `to` keyword is required. See #7 thread for details.

> transform: rotate(0)

Yep, browsers allow omit angle units in some transform functions. But I'm not sure all of them (investigation needed). Because according to [CSS Values and Unit Level 3](https://www.w3.org/TR/css3-values/) omit angle units is prohibited (only length units omitting is allowed), but [CSS Values and Units Module Level 4](https://drafts.csswg.org/css-values/#angles) and [CSS Transform Module Level 1](https://www.w3.org/TR/css-transforms-1/#svg-angle) allow it. So we need confirmation here. I know for sure angle dimensions w/o units doesn't work for gradients in many browsers (or maybe in all of them).

> tap-highlight-color: transparent

Property is non-standard and is not on a standards track. It have no unprefixed version and should be used with `-webkit-` prefix only (see [details on MDN](https://developer.mozilla.org/en/docs/Web/CSS/-webkit-tap-highlight-color)).

> -webkit-print-color-adjust: exact

You're right here – property was missed. Fixed fd463921a67910b138aac755335c814dc74d01d1

> transition: visibility 0 .3s, opacity .3s ease-out

Value is wrong value since `0` should be followed by `ms` or `s` units (check value it in your browser style inspector).

**Comment by a-x-** [comment]
Thank you, @lahmatiy, for the detailed response!
